src/ui/components/comment/comment_nodes.rs

```diff
@@ -9,7 +9,7 @@ pub fn CommentNodes(comments: MaybeSignal<Vec<CommentView>>) -> impl IntoView {
       <For
         each=comments
         key=|cv| cv.comment.id
-        view=move |cv| {
+        children=move |cv| {
             view! {
               <li>
                 <CommentNode comment_view=cv.into()/>
```

Comment:
When you merge main, keep the way main is doing it. Leptos 0.5.0 let's you pass children as children in the markup instead of passing a closure to a component prop.

Reply:
sure, this was just a quick fix as i wasn't focused on this area